src/decoder/mod.rs

```diff
@@ -220,6 +220,30 @@ struct StripDecodeState {
     strip_bytes: Vec<u64>,
 }
 
+#[derive(Debug)]
+struct TileAttributes {
+    // current_tile: usize, // TODO(Tomáš Král): incremental tile decoding
+    tile_offsets: Vec<u64>,
+    tile_bytes: Vec<u64>,
+    tile_width: usize,
+    tile_length: usize,
+    tiles_down: usize,
+    tiles_across: usize,
+    padding_right: usize,    // Length of padding for rightmost tiles in pixels
+    padding_down: usize,     // length of padding for bottommost tile in pixels
+    padding_buffer: Vec<u8>, // A simple buffer right paddding is read into
+    tile_samples: usize,
+    line_samples: usize,       // Sample count of one row of the image
+    row_samples: usize,        // Sample count of one row of one tile
+    tile_strip_samples: usize, // Sample count of one row of tiles
+}
```

Comment:
Comments should be doc-comments above the fields, for example:

```
     /// Length of padding for rightmost tiles in pixels
    padding_right: usize,
```

Reply:
Sure ! I assume the use case for this is running `cargo doc` with `--document-private-items` ?

Reply:
Yep, and because it's just stylistically more common and uniform.